include/rocksdb/db.h

```diff
@@ -818,14 +818,23 @@ class DB {
   // Note that the returned sizes measure file system space usage, so
   // if the user data compresses by a factor of ten, the returned
   // sizes will be one-tenth the size of the corresponding user data size.
-  //
-  // If include_flags defines whether the returned size should include
-  // the recently written data in the mem-tables (if
-  // the mem-table type supports it), data serialized to disk, or both.
-  // include_flags should be of type DB::SizeApproximationFlags
   virtual void GetApproximateSizes(ColumnFamilyHandle* column_family,
                                    const Range* range, int n, uint64_t* sizes,
-                                   uint8_t include_flags = INCLUDE_FILES) = 0;
+                                   const SizeApproximationOptions& options) = 0;
```

Comment:
Option is usually the first argument of the function. We should keep it there too. Since we are changing the function, it's very odd that we don't return Status while there can be underlying I/O error. How about add a return Status to this function?